src/commands/overrides/get.ts

```diff
@@ -107,8 +88,13 @@ export default class DetailedOverrides extends Base {
             return
         }
 
-        this.writer.successMessage(
-            `Override for feature: ${featureKey} on environment: ${environment.key} is variation: ${variation.key}`
-        )
+        this.tableOutput.printOverrides<UserOverride>([{
```

Comment:
why are we not passing in the options with this?

Reply:
its a single entry, so doesn't make sense to pass in the table flags to sort / filter since there's only one item being rendered at most